Swat/SwatActions.php

```diff
@@ -124,7 +124,7 @@ public function init()
 	{
 		parent::init();
 
-		foreach ($this->action_items as $action_item)
+		foreach($this->action_items as $action_item)
```

Comment:
Why take out this space? 
https://github.com/silverorange/coding-standard/blob/master/doc/legacy/Braces.md
http://www.php-fig.org/psr/psr-2/#foreach